tests/system/dcos_service_marathon_tests.py

```diff
@@ -25,8 +24,10 @@ def test_deploy_custom_framework():
     """
 
     client = marathon.create_client()
-    client.add_app(apps.fake_framework())
-    shakedown.deployment_wait(timeout=timedelta(minutes=5).total_seconds())
+    app_def = apps.fake_framework()
+    app_id = app_def["id"]
+    client.add_app(app_def)
+    common.deployment_wait(timeout=timedelta(minutes=5).total_seconds(), service_id=app_id)
```

Comment:
Yeah, I don't think this is correct anymore. This diff changes the timeout to the individual timeout in milliseconds. Here it means the total timeout.

Reply:
good catch